Create popup window for Mobile & Desktop Version

umair-rao opened the pull request:
In this pull request, I have created a popup window for the mobile version & Desktop version.
- The popup window is as per Figma template.
- Popup window iterates data with the help of Javascript for each unique card.
- Popup window closes by clicking on the cross button.
- All the updates as per instructions and Figma template.

umair-rao committed:
Create mobile popup

Shisui6 committed:
Style mobile and desktop popup

umair-rao committed:
Create mobile popup Javascript

Shisui6 committed:
Add javascript to open and close popup

umair-rao committed:
Errors Reomved

mirouhml commented on script.js:
```diff
@@ -21,3 +21,142 @@ cross.addEventListener('click', hideMenu);
+
+// Projects Data to access Dynamically
+
+const projects = [
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    popup_name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the ...
+    image: './images/About1',
+    technologies: ['Ruby Rails', 'Javascript', 'css', 'html'],
+    live_link: '',
+    live_source: '',
+  },
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    popup_name: 'Keeping track of hundreds of components',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the ...
+    image: './images/About1',
+    technologies: ['Ruby Rails', 'Javascript', 'css', 'html'],
+    live_link: '',
+    live_source: '',
+  },
```
- [ ] Please know that the `name` and `popup_name` should be the same because that's the project's name, also for some reason the technologies in the popup screen aren't the ones on the project card:
![image](https://user-images.githubusercontent.com/20567503/195331501-6c0fdf53-3b8e-415e-a18c-c2c542c9d555.png)
![image](https://user-images.githubusercontent.com/20567503/195331506-a61ce974-8563-43c7-9781-a16eee43296a.png)

You should put the same technologies from the objects array you're using, the same goes for the image it should be the same as well, the logic here is that the project cards and the popup both get the information from the objects array, kindly fix these issues and your project should be done! 😁

umair-rao replied:
My template doesn't have images in cards but I added images of portfolio images of the mobile version and desktop version for **Popup window** (to have implementation).

- Alternate image is added for each different card.
- Technologies error is also fixed, now technologies are as per cards. 

Please review the above changes, different images can be added just have to update the path for the **Popup window**.

mirouhml reviewed (changes requested):
# REQUIRED CHANGES 🔴

Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media4.giphy.com/media/KyBesf8kNT6s88SEgb/giphy.gif?cid=ecf05e47n6s5huda7qh40syoppcaspe9vx68ddmxerd08owm&rid=giphy.gif&ct=g)

## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

umair-rao committed:
Fix Images error

umair-rao commented:
My template doesn't have images in cards but I added images of portfolio images of the mobile version and desktop version for **Popup window**.

- Alternate image is added for each different card.
- Technologies error is also fixed, now technologies are as per cards. 

Please review the above changes, different images can be added to the update path for **Popup window**.

1GORDON commented on script.js:
```diff
@@ -21,3 +21,141 @@ cross.addEventListener('click', hideMenu);
+  {
+    name: 'Multi-Post Stories Gain+Glory',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the ...
+    image: '<img src="images/Snapshoot Portfolio.png" alt="Portfolio Image" />',
```
Great job team 🤝

- [ ] [Optional] Just like the previous reviewer requested, kindly make sure the same image displays both in the popup and in the projects section.

1GORDON reviewed (approved):
# Status: APPROVED 🟢 

Hi @umair-rao ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
